skimage/color/tests/test_colorlabel.py

```diff
@@ -198,3 +199,19 @@ def test_negative_intensity():
     labels = np.arange(100).reshape(10, 10)
     image = np.full((10, 10), -1, dtype='float64')
     assert_warns(UserWarning, label2rgb, labels, image, bg_label=-1)
+
+
+def test_bg_color_rgb_string():
+    img = np.random.randint(0, 255, (10, 10), dtype=np.uint8)
+    labels = np.zeros((10, 10), dtype=np.int64)
+    labels[1:3, 1:3] = 1
+    labels[6:9, 6:9] = 2
+    assert_no_warnings(label2rgb, labels, image=img, bg_label=0, bg_color='red')
```

Comment:
would you mind making the test more specific, like for example testing that one of the background pixels is indeed red? (if it's not too complicated)

Reply:
hmm, I am not sure it's so easy to test because the background pixels isn't really red in the `overlay` mode because of https://github.com/scikit-image/scikit-image/blob/c9151096cf095afb3811c8e27d491e52f989e5fb/skimage/color/colorlabel.py#L197

Or do you think I should plug in the defaults and checked that the background pixel matches the output of that calculation referenced ^^?

Reply:
```suggestion
    output = label2rgb(labels, image=img, bg_label=0, bg_color='red', alpha=0.9)
    assert output[0, 0, 0] > 200  # red channel
```

Reply:
increasing `alpha` makes it easier to test the red color :-)